omnigibson/simulator.py

```diff
@@ -767,6 +770,7 @@ def step_physics(self):
             """
             Step the physics a single step.
             """
+            PoseAPI.invalidate()
```

Comment:
on a high-level note, for all these `invalidate` call, why not just call `refresh` directly? 

also, nit, this line should be after `self._physics_context._step(current_time=self.current_time)`?